### DISCUSSION TITLE
[BUG] Статус полного узла Не синхронизирован

### DISCUSSION THREAD

**Discussion by alexJew322** [Chia-Network/chia-blockchain#3266] [Support]
Перерыл кучу форумов, но ничего кроме как открыть порт 8444 не нашел, но он у меня по дефолту открыт, все было норм, с 25го числа висит статус не синхронизирован и все,ничего не помогает, есть решение какое то?, переустановка,перезапуски компа и тд и тп ничего не дают. Мб просто ждать некст обновления?
![d7SErac1KQA](https://user-images.githubusercontent.com/83288564/116247717-cc900400-a773-11eb-9ea9-0b9e45964627.jpg)

**Comment by Mlalosevic** [comment]
What version you are running? add IP adresses as much as you can remember it will speed the sync. Do not panic if the wallet sync later on

**Comment by baybekovtimur** [comment]
Решил проблему синхронизации с помощью NO-IP.

**Comment by alexJew322** [comment]
> Решил проблему синхронизации с помощью NO-IP.

можно по подробнее

**Comment by baybekovtimur** [comment]
> > Решил проблему синхронизации с помощью NO-IP.
> 
> можно по подробнее

Пробросил порт 8444 на роутере не помогло. потом зарегистрировался на сайте No IP установил программу DUC.  Ввел туда учетную запись No IP. Потом на сайте No IP в меню No IP hostname изменил белый IP адрес на локальный IP. в программе CHIA добавил node-eu.chia.net;8444 после этого пошла синхронизация.
![Безымянный](https://user-images.githubusercontent.com/83324835/116438932-65a04700-a868-11eb-8b9a-ac9048655cc5.png)

**Comment by baybekovtimur** [comment]
![Uploading Безымянный.png…]()

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
